cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -447,15 +446,16 @@ def test_upload_with_non_ascii_charaters(self, mock_conn):
         response = json.loads(response.content.decode('utf-8'))
         self.assertEqual(response['error'], u'The file name for %s must contain only ASCII characters.' % file_name)
 
+    @override_settings(AWS_ACCESS_KEY_ID='test_key_id', AWS_SECRET_ACCESS_KEY='test_secret', AWS_SECURITY_TOKEN='token')
     @patch('boto.s3.key.Key')
     @patch('boto.s3.connection.S3Connection')
     @override_flag(waffle_flags()[ENABLE_DEVSTACK_VIDEO_UPLOADS].namespaced_flag_name, active=True)
-    def test_assume_role_connection(self, mock_conn, mock_key):
+    def test_devstack_upload_connection(self, mock_conn, mock_key):
         files = [{'file_name': 'first.mp4', 'content_type': 'video/mp4'}]
         credentials = {
-            'access_key': 'test_key',
+            'access_key': 'test_key_id',
```

Comment:
Can we do something like `access_key: settings. AWS_SECRET_ACCESS_KEY`, to avoid the dup strings?